titan-server/src/schema.rs

```diff
@@ -33,11 +33,14 @@ table! {
         // disableSignature -> Bool,
         // disableSignatureReason -> Nullable<Text>,
         // disableSignatureExpires -> Integer,
-        // lastActivityTime -> Integer,
+        lastActivityTime -> Integer,
         // profileHits -> Integer,
         // rankID -> Nullable<Integer>,
-        // userTitle -> Varchar,
-        // userOnlineGroupID -> Nullable<Integer>,
+        #[sql_name = "userTitle"]
+        user_title -> Varchar,
```

Comment:
Out of curiosity, @MichaelJamesParsons, what's the reason for deviating the name in the `schema!` macro from the actual column name in the database? I didn't think this threw a warning on compilation.